lingpy/sequence/ngrams.py

```diff
@@ -0,0 +1,685 @@
+# *-* coding: utf-8 *-*
+"""
+This module provides various methods for generating and
```

Comment:
Note that the documentation requires one single line when introducing a module. So make this a one liner that introduces the module (This moodule provides various methods for generating ngrams) and then add a blank line, and add the more detailed information.